lib/ansible/modules/cloud/vmware/vmware_guest_snapshot_info.py

```diff
@@ -67,20 +68,20 @@
 '''
 
 EXAMPLES = '''
-  - name: Gather snapshot facts about the virtual machine in the given vCenter
-    vmware_guest_snapshot_facts:
+  - name: Gather snapshot info about the virtual machine in the given vCenter
+    vmware_guest_snapshot_info:
       hostname: "{{ vcenter_hostname }}"
       username: "{{ vcenter_username }}"
       password: "{{ vcenter_password }}"
       datacenter: "{{ datacenter_name }}"
       name: "{{ guest_name }}"
     delegate_to: localhost
-    register: snapshot_facts
+    register: snapshot_info
 '''
 
 RETURN = """
 guest_snapshots:
-    description: metadata about the snapshot facts
+    description: metadata about the snapshot information
```

Comment:
Same here.

Reply:
Nothing to change here right since the return variable name doesn't have any naming convention such as "guest_snapshots_facts" or "guest_snapshots_info". So, I guess we could keep it as it is

@felixfontein : Your view/suggestion on this?

Reply:
@felixfontein I got your point after observing the returned values

Reply:
Fixing this as well

Reply:
Ah, sorry, I apparently got confused when looking at too many diffs :) This one is fine!

Reply:
Cool, thanks